client/src/plus/collections-quicksearch.ts

```diff
@@ -38,22 +40,27 @@ function setCollectionItemsUpdatedDate(updated: Date) {
   }
 }
 
-export function getCollectionItems(): CollectionItem[] {
+export async function getCollectionItems(
```

Comment:
Does this need to be `async`? We call an async function (`fetchAllCollectionsItems`), but we don't `await` it, should we?

Reply:
the `await` was missing in the next line. Good catch.